tests/test_handler.py

```diff
@@ -13,26 +14,118 @@ def test_handle(self):
+            f'pysubyt --force  --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "iteration"',  # noqa
+            f'pysubyt --force  --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "MODE"',  # noqa
```

Comment:
I would try to change this to some sort of function to generate those.

And if you really wanted them hardcoded then on a different file then the test

Reply:
Hmm, then how do we test that function?

If those strings were reused somewhere else, I would place them in a separate file, but for now I prefer to keep them in the `test_handler.py` file.

Reply:
It could be a functions that return this as a list.
To test this you could test if the different variables appears the number they are supposed to inside the list

Reply:
It could be multiple functions. 
Since not all entries have the same number of variables

Reply:
I don't see it :D So `PysubytHandler().handle` generates these strings from a list of tasks, and your suggestions would be to write a function that again generates these strings from a list of tasks and then compare the strings. Isn't that just reimplementing `PysubytHandler().handle`?